cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -104,7 +107,7 @@ endif()
 
 set(_cuda_compile_flags " ")
 if (ENABLE_CLANG_CUDA)
-    set (_cuda_compile_flags "-x cuda --cuda-gpu-arch=${BLT_CLANG_CUDA_ARCH} --cuda-path=${CUDA_TOOLKIT_ROOT_DIR}")
+    set (_cuda_compile_flags "-x;cuda;--cuda-gpu-arch=${BLT_CLANG_CUDA_ARCH};--cuda-path=${CUDA_TOOLKIT_ROOT_DIR}")
```

Comment:
I think you should be able to accomplish the same by removing the outer quotation marks.
I.e.
```cmake
     set (_cuda_compile_flags -x cuda --cuda-gpu-arch=${BLT_CLANG_CUDA_ARCH} --cuda-path=${CUDA_TOOLKIT_ROOT_DIR}) 
```

(either way should be fine though)

Reply:
Good suggestion, I switched it.